examples/jupyter/tensor_addition.ipynb

```diff
@@ -0,0 +1,226 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "866d36f8",
+   "metadata": {},
+   "source": [
+    "### Installation"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "fa32231a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# clone the wave github repo\n",
+    "!git clone https://github.com/iree-org/wave.git\n",
+    "# install ROCm Pytorch dependencies\n",
+    "!pip install -r pytorch-rocm-requirements.txt\n",
+    "# install wave and its dependencies\n",
+    "!pip install wave-lang"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "52993703",
+   "metadata": {},
+   "source": [
+    "### Wave Tensor/Matrix Addition Kernel Tutorial"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e55e18af",
+   "metadata": {},
+   "source": [
+    "#### (1) Importing Libraries"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b636039f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import torch\n",
+    "\n",
```

Comment:
Can we import as wave? (tkw was legacy and I will submit a PR this weekend to rename all tkw imports to wave)